src/libcore/raw.rs

```diff
@@ -30,9 +63,13 @@ pub struct Slice<T> {
 
 impl<T> Copy for Slice<T> {}
 
-/// The representation of a Rust closure
+/// The representation of an old Rust closure
 #[repr(C)]
 #[derive(Copy)]
+#[unstable(feature = "core")]
```

Comment:
Yup, the compiler told me to (something like "deprecated must be used in conjunction with unstable or stable").